File: 151002/Ryabov/DC/DC.Api/Models/Creator.cs
using System.ComponentModel.DataAnnotations;
using Microsoft.EntityFrameworkCore;

namespace Forum.Api.Models;

public class Creator
{
    public long Id { get; set; }
    
    public string Login { get; set; }
    
    public string Password { get; set; }
    
    public string FirstName { get; set; }
    
    public string LastName { get; set; }
}

File: 151002/Naftolski/Lab2/MessageApp/src/main/java/by/bsuir/messageapp/model/entity/Creator.java
package by.bsuir.messageapp.model.entity;

import jakarta.persistence.*;
import lombok.Getter;
import lombok.Setter;
import org.springframework.boot.autoconfigure.domain.EntityScan;

import java.util.ArrayList;
import java.util.List;

@Getter
@Setter
@Entity
@Table(name = "tbl_creator")
public class Creator {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    @Column(name = "id", nullable = false)
    private Long id;

    @Column(name = "login", unique = true, nullable = false, length = 64)
    private String login;

    @Column(name = "password", length = 128)
    private String password;

    @Column(name = "firstname", length = 64)
    private String firstName;

    @Column(name = "lastname", length = 64)
    private String lastName;

    @OneToMany(mappedBy = "creator", cascade = CascadeType.ALL)
    private List<Story> stories = new ArrayList<>();
}
